src/main/kotlin/blackjack/controller/Controller.kt

```diff
@@ -0,0 +1,98 @@
+package blackjack.controller
+
+import blackjack.model.CardDeck
+import blackjack.model.Dealer
+import blackjack.model.ErrorMessage
+import blackjack.model.Player
+import blackjack.model.Players
+import blackjack.model.ResultEvaluator
+import blackjack.view.InputView
+import blackjack.view.OutputView
+import blackjack.view.OutputView.displayInitialCardsMessage
+
+class Controller {
+    private val deck = CardDeck()
+    private var _players: Players? = null
+    private val dealer = Dealer()
+
+    private val players: Players
+            get() = _players!!
+
+    fun runGame() {
+        _players = initializePlayers()
+        OutputView.displayPlayerNames(players)
+        drawInitialCards()
+        OutputView.displayFirstCardMessage(dealer)
+        players.forEach { OutputView.displayAllCardsMessage(it) }
+        players.dealCards()
+        dealer.dealCards()
+        displayCardsAndTotal()
+        printResults()
+    }
+
+    fun printResults() {
```

Comment:
```suggestion
    private fun printResults() {
```

Let's keep methods as private if they aren't used by other classes.
Please check other methods across the whole project as well.

Reply:
Thanks for the suggestion! I made the `printResults()` method and a lot of other methods private. I also enabled visibility inspections in IntelliJ to help catch similar cases automatically and keep the code cleaner by restricting method visibility whenever possible.